llvm-external-projects/iree-dialects/BUILD

```
@@ -820,13 +820,20 @@ cc_library(
         "test/lib/**/*.cpp",
     ]),
     deps = [
+        "IREELinalgExtTransforms",
```

I think they are both correct. I'd like to keep things consistent.

```suggestion
        ":IREELinalgExtTransforms",
```